src/chainparams.cpp

```diff
@@ -42,7 +42,6 @@ class CMainParams : public CChainParams {
         consensus.nMajorityEnforceBlockUpgrade = 750;
         consensus.nMajorityRejectBlockOutdated = 950;
         consensus.nMajorityWindow = 4000;
-        // TODO generate harder genesis block
```

Comment:
I was leaving these in since we may generate an even harder genesis block for 1.0.0, but does not block.